network/network_if_mbedos.c

```diff
@@ -203,8 +209,9 @@ fdo_con_handle fdo_con_connect(fdo_ip_address_t *ip_addr, uint16_t port,
  */
 int32_t fdo_con_disconnect(fdo_con_handle handle, void *ssl)
 {
-	if (ssl) // SSL disconnect
+	if (ssl) {// SSL disconnect
```

Comment:
nitpick - should there be a space between the brace and comment (or comment on separate line)